edit_topictagged_form.php

```diff
@@ -75,7 +75,7 @@ protected function definition_inner($mform) {
 
 	// Remove all difficulty and last_used tags from the list
 	foreach ($tagstrings as $standardtag) {
-		if (strpos($standardtag, "last_used") !== false)
+		if (strpos($standardtag, "last_used") !== false || strpos($standardtag, "###__###") !== false)
```

Comment:
For this, using [str_contains](https://www.php.net/manual/en/function.str-contains.php) may be easier to read. It always returns a bool.

Reply:
From what I see, the str_contains function works for php 8.
I would keep using the strpos function to keep the extended compatibility. What do you think?

Reply:
Fair enough, didn't see the php 8.0 mention in the page. Keep this as-is for now. At some point Moodle bump the minimum php requirement, but as long as php 7.2 is the minimum supported version for 3.10, the code must be implemented using compatible functions.